game.py

```diff
@@ -20,8 +20,34 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses  = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
 
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES  :
+
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word :
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses [user_input] =True
```

Comment:
```suggestion
            correct_letter_guess_statuses[user_input] = True
```

There is an extra white space before `[`. When you set a key/value pair in a dictionary it should look like dict[key] = value` with no extra or missing whitespaces.

There is a missing whitespace after `=`.

Take care to look over your own code before you submit it for review, especially when you're at internship. Small inconsistencies in a large codebase are like a death by 1000 paper cuts. Each inconsistency makes the codebase more difficult to maintain.